examples/kubernetes/kafka-oauth-single-authz-service-accounts.yaml

```diff
@@ -0,0 +1,78 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: oidc-reader
+  annotations:
+    kubernetes.io/description: |
+        A cluster role which allows access to the OpenID Connect endpoints on
+        the API Server.
+rules:
+  - nonResourceURLs: ["/.well-known/openid-configuration", "/openid/v1/jwks"]
+    verbs: ["get", "post"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: oidc-reader-binding
+  annotations:
+  kubernetes.io/description: |
```

Comment:
Still not ok. It's this line (19) that should be indented in order to be a child of `annotations`. Currently it is a child of `metadata` which is incorrect.

Reply:
Based off of @scholzj's feedback, I've backed this example out and will open up a subsequent PR with a better "out of the box" experience around service account usage.